zeppelin-server/src/test/java/org/apache/zeppelin/rest/AbstractTestRestApi.java

```diff
@@ -315,6 +315,10 @@ private static boolean isActiveSparkHome(File dir) {
   }
 
   protected static void shutDown() throws Exception {
+    shutDown(true);
+  }
+
+  protected static void shutDown(final boolean deleteConfDir) throws Exception {
```

Comment:
What's the purpose for this change ? whether delete conf is controlled by `ZeppelinConfiguration`, it seems not necessary to be called by user explicitly.

Reply:
I did not see another proper chance to avoid deletion of intepreter.json file from the newly added test case. the test case needs to restart the server, which would delete the interpreter bindings without that change.

Reply:
I see